packages/wrangler/src/__tests__/versions/versions.upload.test.ts

```diff
@@ -98,8 +111,74 @@ describe("versions upload", () => {
 			 \\"abc\\": \\"def\\",
 			 \\"bool\\": true
 			}
+			Uploaded test-worker (TIMINGS)
+			Worker Version ID: 51e4886e-2db7-4900-8d38-fbfecfeab993"
+		`);
+	});
+
+	test("should print preview url if version has preview", async () => {
+		mockGetScript();
+		mockUploadVersion(true);
+		mockGetWorkerSubdomain(true);
+		mockSubDomainRequest();
+
+		// Setup
+		fs.mkdirSync("./versions-upload-test-worker", { recursive: true });
```

Comment:
what's this for? there is already a `runInTmpDir()` for these tests

Reply:
I removed the redundant mkdirSync and absorbed it into the test refactor commit.